crates/sui-storage/src/event_store/test_utils.rs

```diff
@@ -47,12 +47,13 @@ impl TestEvent {
 
 pub fn new_test_publish_event(
     timestamp: u64,
+    digest: TransactionDigest,
     seq_num: u64,
     sender: Option<SuiAddress>,
 ) -> EventEnvelope {
     EventEnvelope::new(
         timestamp,
-        None,
+        Some(digest),
```

Comment:
can we just generate the TX digest here?   It's just random in all the test code, this would make the test code a bit more compact and make your Diff smaller. :)

Reply:
The reason i changed it was because the test code was previously inserting events with identical sequence numbers but different digests. That was nonsensical, and stopped working after aggregation by tx digest, so I move the digest creation out of here, so that it would be possible to create two events with the same tx digest.